frontend/package.json

```diff
@@ -20,10 +20,16 @@
     "@types/react": "^18.3.18",
     "@types/react-dom": "^18.3.5",
     "@vitejs/plugin-react": "^4.3.4",
+    "autoprefixer": "^10.4.20",
     "eslint": "^9.17.0",
     "eslint-plugin-react-hooks": "^5.0.0",
     "eslint-plugin-react-refresh": "^0.4.16",
     "globals": "^15.14.0",
+    "init": "^0.1.2",
```

Comment:
The frontend stuff needs moving to a separate PR, because it's out of ticket scope :-)